wolfssl/wolfcrypt/types.h

```diff
@@ -512,15 +512,15 @@ decouple library dependencies with standard string, memory and so on.
             DECLARE_ARRAY(VAR_NAME, VAR_TYPE, VAR_ITEMS, VAR_SIZE, HEAP)
         #define DECLARE_ARRAY_DYNAMIC_EXE(VAR_NAME, VAR_TYPE, VAR_ITEMS, VAR_SIZE, HEAP)
         #define FREE_ARRAY_DYNAMIC(VAR_NAME, VAR_ITEMS, HEAP) \
-            FREE_ARRAY(VAR_NAME, VAR_ITEMS, HEAP)
+            WC_FREE_ARRAY(VAR_NAME, VAR_ITEMS, HEAP)
     #else
         #undef DECLARE_VAR_IS_HEAP_ALLOC
         #define DECLARE_VAR(VAR_NAME, VAR_TYPE, VAR_SIZE, HEAP) \
             VAR_TYPE VAR_NAME[VAR_SIZE]
         #define DECLARE_ARRAY(VAR_NAME, VAR_TYPE, VAR_ITEMS, VAR_SIZE, HEAP) \
             VAR_TYPE VAR_NAME[VAR_ITEMS][VAR_SIZE]
         #define FREE_VAR(VAR_NAME, HEAP) /* nothing to free, its stack */
```

Comment:
Can you also refactor the `FREE_VAR` -> `WC_FREE_VAR`? Maybe the others too? This is an internal macro used for testing.

Reply:
Refactored remaining defines as well.

Reply:
This unfortunately causes errors in FIPS code that is set in stone. I'll add FIPS exceptions.